remix/app/services/auth.server.ts

```diff
@@ -0,0 +1,44 @@
+import { Authenticator } from "remix-auth";
+import { sessionStorage } from "./session.server";
+import { GoogleStrategy, SocialsProvider } from "remix-auth-socials";
+import type { UserProfile } from "~/types";
+import { db } from "~/utils/db.server";
+
+export const authenticator = new Authenticator(sessionStorage);
+
+async function handleSocialAuthCallBack({ profile }: any) {
+  const userProfile: UserProfile = profile?._json;
```

Comment:
Add a line breaks between variable definition and code logic locks for readability:

```suggestion
  const userProfile: UserProfile = profile?._json;
  
  if (userProfile) {
```